index.html

```diff
@@ -28,16 +28,148 @@
     <section class="headline-flex">
         <div class="headline-heading-placeholder">
             <h1 class="headline-heading-name">Gabriel Fonseca</h1>
-            <h2 class="headline-heading-occupation">Software Developer</h2>
+            <h3 class="headline-heading-occupation">Software Developer</h3>
         </div>
         <div class="headline-description-placeholder">
             <p class="headline-description">I can help you build a product, feature or website. Look through some of my work and experience! If you like what you see and have a project you need coded, don't hesitate to contact me.</p>
         </div>
     </section>
+
+    <div class="section-separation"></div>
+
+    <div class="about-me-flex">
+      <div class="about-me-languages">
+        <div class="about-me-title">Language
+          <ul class="about-me-list">
+            <li class="about-me-list-item">Javascript</li>
+            <li class="about-me-list-item">Ruby</li>
+            <li class="about-me-list-item">Html</li>
+            <li class="about-me-list-item">Css</li>
+         </ul>
+      </div>
+      </div>
+      <div class="about-me-frameworks">
+        <div class="about-me-title">Frameworks
+          <ul class="about-me-list">
+            <li class="about-me-list-item">Bootstrap</li>
+            <li class="about-me-list-item">Ruby on Rails</li>
+            <li class="about-me-list-item">RSPec</li>
+            <li class="about-me-list-item">CapyBara</li>
+            <li class="about-me-list-item">Selenium</li>
+          </ul>
+        </div>
+      </div>
+      <div class="about-me-skills">
+        <div class="about-me-title">Skills
+          <ul class="about-me-list">
+          <li class="about-me-list-item">Codekit</li>
+          <li class="about-me-list-item">Github</li>
+          <li class="about-me-list-item">Codepen</li>
+          <li class="about-me-list-item">Gitlab</li>
+          <li class="about-me-list-item">Terminal</li>
+          </ul>
+        </div>
+      </div>
+    </div>
+
+    <div class="section-separation"></div>
+      <div class="contact-icons">
+        <a href="https://github.com/Gopxfs" class="contact-github">
+          <img class="github-icon" src="/images/github-icon.png" alt="GitHub Logo">   
+        </a>
+        <a href="https://www.linkedin.com/in/gabriel-fonseca-sales-8bb64b236/" class="contact-linkedin">
+          <img class="linkedin-icon" src="/images/linkedin-logo.png" alt="LinkedIn Logo">
+        </a>
+        <a href="https://twitter.com/FonsecaSGabe" class="contact-twitter">
+          <img class="twitter-icon" src="/images/twitter-logo.png" alt="Twitter Logo">
+        </a>
+      </div>
```

Comment:
- [ ] Contact icons are almost ready! Let's use `ul` and `li` tags to list them as this is a better practice for listings in HTML.